receiver/dcgmreceiver/factory_others.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !windows
-// +build !windows
+//go:build !windows && !darwin
```

Comment:
Maybe we can just make this exclusive to Linux and change the corresponding file names to match?

Reply:
Good suggestion - done!